components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/LocalAndOutboundAuthenticationConfig.java

```diff
@@ -89,6 +91,12 @@ public class LocalAndOutboundAuthenticationConfig implements Serializable {
     @XmlElement(name = SKIP_LOGOUT_CONSENT)
     private boolean skipLogoutConsent = false;
 
+    @XmlElement(name = USE_EXTERNAL_CONSENT_MANAGEMENT)
+    private boolean useExternalConsentManagement = false;
+
+    @XmlElement(name = EXTERNAL_CONSENT_URL)
+    private String externalConsentUrl;
```

Comment:
would it be better to define a model object related to consent here instead of individual properties?

Reply:
will check and update

Reply:
Resolved by https://github.com/wso2/carbon-identity-framework/pull/4475/commits/e85ffa252aaf5f347ac8f57b041e81b8dd6e99c5